Mathlib/Analysis/LocallyConvex/Polar.lean

```diff
@@ -64,6 +64,10 @@ theorem polar_mem (s : Set E) (y : F) (hy : y ∈ B.polar s) : ∀ x ∈ s, ‖B
 theorem zero_mem_polar (s : Set E) : (0 : F) ∈ B.polar s := fun _ _ => by
   simp only [map_zero, norm_zero, zero_le_one]
 
+instance (s : Set E) : Nonempty (B.polar s) := by
```

Comment:
Since `polar _ _` is a set, we usually use a theorem claiming `Set.Nonempty`, not an instance claiming `_root_.Nonempty`. Do you need the instance for some reason?

Reply:
@urkud currently I'm using it here: https://github.com/leanprover-community/mathlib4/pull/16316/files#diff-b605c62b370366aa8783f5424b695ad108ee3f82e0f878dc48c9c3ade63cd8c2R343

Maybe there's another way to do that though?

Reply:
I've fixed #16316 using `haveI` (for now at least). I've made the instances in this PR theorems instead.